segmentation_labeling_app/utils/query_utils.py

```diff
@@ -54,4 +56,4 @@ def query(query, user, host, database, password, port):
     finally:
         cursor.close()
         conn.close()
-    return results
\ No newline at end of file
+    return results
```

Comment:
Again I think there should be a new line delimiter at the end of the file

Reply:
I think if you try it, you will see that there is a line delimiter at the end of the file, there is not a blank line though. These are different.

Reply:
Right that's my bad I didn't see the delimiter.